ethereum/core/src/main/java/org/hyperledger/besu/ethereum/transaction/TransactionSimulator.java

```diff
@@ -236,7 +236,7 @@ public Optional<TransactionSimulatorResult> processWithWorldUpdater(
             : blockHeaderToProcess.getGasLimit();
     if (rpcGasCap > 0) {
       gasLimit = rpcGasCap;
-      LOG.info("Capping gasLimit to " + rpcGasCap);
+      LOG.atTrace().setMessage("Capping gasLimit to {}").addArgument(rpcGasCap).log();
```

Comment:
can we specify that it is related only to transaction simulation ?

Reply:
updated the log